vgrep/PKGBUILD

```diff
@@ -1,35 +1,39 @@
 # Maintainer: acxz <akashpatel2008 at yahoo dot com>
 # Contributor: Darshit Shah <darnir@gmail.com>
+# Contributor: Jonas Malaco <jonas@protocubo.io>
 
 pkgname=vgrep
 pkgver=2.4.0
 pkgrel=1
 pkgdesc="Reimpementation of the ancient cgvg perl scripts"
-arch=('x86_64' 'i686')
-license=('GPLv3')
+arch=('x86_64')
+license=('GPL3')
 depends=('go')
 url="https://github.com/vrothberg/vgrep"
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/vrothberg/${pkgname}/archive/v${pkgver}.tar.gz")
 sha256sums=('d0dd594156638158f7163d37169dd02e3a82eabfa6c8f57d6eb1dfa89c669c03')
 
 prepare() {
-  mkdir $srcdir/{bin,pkg,src}
-  mv $srcdir/${pkgname}-${pkgver}/ $srcdir/src/
-  cd $srcdir/src/$pkgname-$pkgver
+  cd "$pkgname-$pkgver"
+  mkdir -p build/
```

Comment:
Can we just move this into the `build()` function, no need to have a `prepare()` function that just creates a build folder.

Reply:
I agree with you.  In fact, I think the guidelines are wrong to suggest otherwise:

> If extraction is skipped (makepkg --noextract), then prepare() is not run.